spec/03-plugins/01-tcp-log/01-tcp-log_spec.lua

```diff
@@ -435,7 +435,7 @@ for _, strategy in helpers.each_strategy() do
       local log_message = cjson.decode(res)
 
       assert.equal("grpcs", log_message.service.protocol)
-      assert.equal("TLSv1.2", log_message.request.tls.version)
+      assert.equal("TLSv1.3", log_message.request.tls.version)
```

Comment:
unsure why this is needed, but i suspect go / go grpc changed its default tls version